### PR TITLE
gui-apps/tuigreet: added git-live 9999 version

### PR DESCRIPTION
**Pull request by DjLogozzo**
Changes:
* Added git-live tuigreet-9999.ebuild

Package-Manager: Portage-3.0.28, Repoman-3.0.3
Signed-off-by: Dee-Jay Logozzo <gentoo@djl.id.au>

### CODE REVIEW

File: gui-apps/tuigreet/tuigreet-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+"
+
+inherit cargo
+
+DESCRIPTION="TUI greeter for greetd login manager"
+HOMEPAGE="https://github.com/apognu/tuigreet"
+
+if [ ${PV} == "9999" ] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/apognu/${PN}"
```
Comment (juippis) [inline gui-apps/tuigreet/tuigreet-9999.ebuild:16]: In future, 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D_in_SRC_URI
please.

File: gui-apps/tuigreet/tuigreet-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+else
+	SRC_URI="https://github.com/apognu/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
```
Comment (juippis) [inline gui-apps/tuigreet/tuigreet-9999.ebuild:18]: Same here for the PN part.

**Review by juippis — APPROVED** [review]
I am somewhat reluctant in adding live ebuilds for unmaintained ebuilds... but it's nice to see Gentoo can be used for upstream development, so I support it here. Thanks!

PS I've strip out your sign-off even previously because it doesn't seem like it's your real name. Please if you don't use your real name, don't add sign-off at all. It's better for us that way. 
If that is your real name, I apologize for my actions - just playing it safe.

**Comment by DjLogozzo** [comment]
Hey @juippis
Thanks for approving my PR, I'm glad to be able to contribute back to the community.

> PS I've strip out your sign-off even previously because it doesn't seem like it's your real name. Please if you don't use your real name, don't add sign-off at all. It's better for us that way. 
> If that is your real name, I apologize for my actions - just playing it safe.
 
Dee-Jay is my real name though I completely understand your hesitancy in including it, it is a fairly rare name and quite common as a nickname, if you'd like I can provide proof that it is my legal name or if you have any suggestions on what else I can do to help the situation

**Comment by juippis** [comment]
I trust you. Thank you very much for replying!

**Pull request closed without merging**